build.gradle.kts

```diff
@@ -19,7 +20,7 @@ plugins {
 
 java {
     toolchain {
-        languageVersion = JavaLanguageVersion.of(11)
+        languageVersion = JavaLanguageVersion.of(17)
```

Comment:
This is the key of my commits. This change forces all Gradle processes run on Java 17.

Reply:
The built class files still formatted for Java 8, so we do not have to ship a new release by this PR.